.vscode/notebooks/CITestResults.ipynb

```diff
@@ -295,12 +334,12 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.13"
+   "version": "3.9.6"
   },
   "orig_nbformat": 4,
   "vscode": {
    "interpreter": {
-    "hash": "05cadb55cbb23d0d156eeac1bfe39405b391b4fa73e06b1a6140e7007eb56649"
+    "hash": "33193fed03cec9a59993558aba41e5916503fab5abae601cbbc0cdfec3d419dd"
```

Comment:
This probably should be transient metadata.

Reply:
no it can't, we use this to reselect the right interpreter/kernel
once ian makes ALL necessary changes to kernel auto selection, we can review removing this completely.
@IanMatthewHuff /cc

i think it's a hack to store this in metadata, only works for a specific machine, hence not transferable to other machines.
besides we can track last selected kernel other ways without storing metadata in the nb.

Reply:
agree that it should go away completely (`vscode` section) otherwise there is always dirty change when the notebook is being worked on collaboratively.